Sources/SwiftRobot/Protocols/RobotTaskBuilder.swift

```diff
@@ -14,6 +14,6 @@ public enum RobotTaskBuilder {
     ) -> RobotTask { task }
     
     public static func buildBlock(
-        _ tasks: any RobotTask...
-    ) -> [any RobotTask] { tasks }
+        _ tasks: RobotTask...
+    ) -> [RobotTask] { tasks }
```

Comment:
RobotTask doesn't have an associatedtype anymore so we don't need to use `any` to wrap it.